<launch>
  <node pkg="safety_controller" name="safety_controller" type="safety_controller_node" respawn="true" output="screen">
    <rosparam file="$(find safety_controller)/launch/safety_controller.yaml" command="load" />

    <remap from="teleop_cmd_vel" to="plan_cmd_vel" />
  </node>
</launch>
